bedita-app/controllers/components/api_validator.php

```diff
@@ -374,6 +374,20 @@ public function isObjectReachable($objectId, $checkPermissions = true) {
         if (!$this->enableObjectReachableCheck) {
             return true;
         }
+        // User card always reachable
```

Comment:
To avoid repeating the query every time the method is called maybe we could move the Card load in `ApiAuthComponent::findUser()` but I think in `$user` var you should already have data about `ObjectUser`.